google/cloud/forseti/common/data_access/violation_map.py

```diff
@@ -38,6 +38,7 @@
     'FIREWALL_WHITELIST_VIOLATION': 'firewall_rule_violations',
     'GROUP_VIOLATION': 'groups_violations',
     'KE_VERSION_VIOLATION': 'ke_version_violations',
+    'KE_JMESPATH_VIOLATION': 'ke_jmespath_violations',
```

Comment:
Can we just name this as `KE_VIOLATION`?  jmespath is just what we used to evaluate for the violation.

Reply:
Yes - pushing a fix to this and all of the other "jmespath" naming issues